crates/red_knot_module_resolver/src/path.rs

```diff
@@ -1,12 +1,11 @@
+use ruff_db::files::{system_path_to_file, File};
+use ruff_db::system::{SystemPath, SystemPathBuf};
```

Comment:
Nit: I'd prefer these imports to be below the module docstring and below the standard-library imports

Reply:
Makes sense, but this is not a module level comment. I'll fix it.